### ISSUE TITLE
example1 & example2 included in default blacklist

### ISSUE THREAD

**Issue by jakevdp** [pypa/bandersnatch#95]
I've found that if you don't specify a blacklist in ``bandersnatch.conf``, the ``bandersnatch mirror`` command acts as if you have included the example blacklist from the documentation.

here is my configuration file at ``/etc/bandersnatch.conf``:
```
[mirror]

; The directory where the mirror data will be stored.
directory = /srv/pypi

; Save JSON metadata into the web tree:
json = false

; The PyPI server which will be mirrored.
master = https://pypi.org

; The network socket timeout to use for all connections.
timeout = 10

; Number of worker threads to use for parallel downloads.
workers = 3

; Whether to hash package indexes
; Recommended setting: the default of false for full pip/pypi compatibility.
hash-index = false

; Whether to stop a sync quickly after an error is found or whether to continue
; syncing but not marking the sync as successful. Value should be "true" or
; "false".
stop-on-error = false

; Number of consumers which verify metadata
verifiers = 3
```
And here is the beginning of the output:
```
$ bandersnatch mirror
2018-11-26 22:41:07,145 INFO: bandersnatch/3.1.1 (cpython 3.6.7-final0, Linux x86_64)
2018-11-26 22:41:07,146 INFO: Setting up mirror directory: /srv/pypi/
2018-11-26 22:41:07,146 INFO: Setting up mirror directory: /srv/pypi/web/simple
2018-11-26 22:41:07,147 INFO: Setting up mirror directory: /srv/pypi/web/packages
2018-11-26 22:41:07,147 INFO: Setting up mirror directory: /srv/pypi/web/local-stats/days
2018-11-26 22:41:07,147 INFO: Generation file missing. Reinitialising status files.
2018-11-26 22:41:07,148 INFO: Status file missing. Starting over.
2018-11-26 22:41:07,148 INFO: Syncing with https://pypi.org.
2018-11-26 22:41:07,148 INFO: Current mirror serial: 0
2018-11-26 22:41:07,148 INFO: Syncing all packages.
2018-11-26 22:41:11,704 INFO: Package 'example2' is blacklisted
2018-11-26 22:41:11,704 ERROR: example2 not found in packages to sync - blacklist_project has no effect here ...
2018-11-26 22:41:11,949 INFO: Package 'example1' is blacklisted
2018-11-26 22:41:11,949 ERROR: example1 not found in packages to sync - blacklist_project has no effect here ...
...
```
Note the mention of `example1` and `example2`, which was unexpected.

**Comment by cooperlees** [comment]
Thanks for the report. Will try track it down, if anyone else does, please send the PR!

**Comment by jakevdp** [comment]
It looks like the issue is here: 

https://github.com/pypa/bandersnatch/blob/422770208f70fac0d9b67e85f1d29f0ffabc3e5f/src/bandersnatch/configuration.py#L32-L34

The default config file is always loaded, and it includes ``example1`` and ``example2`` in the blacklist.

Perhaps the project should split this file into two: one to act as the user-facing default (including the blacklist example) and one to act as the internal default (without the blacklist example).

**Comment by GreatBahram** [comment]
I fixed this on PR #96

**Comment by cooperlees** [comment]
Fix going out in 3.1.2
